lib/hnvm/ensure_bin.sh

```diff
@@ -71,11 +71,11 @@ function download_pnpm() {
   blue "Downloading pnpm v${pnpm_ver} to ${HNVM_PATH}/pnpm" > "${COMMAND_OUTPUT}"
 
   if [[ "${HNVM_QUIET}" == "true" ]]; then
-    curl -L https://raw.githubusercontent.com/pnpm/self-installer/master/install.js --silent |
+    curl -n -L https://raw.githubusercontent.com/pnpm/self-installer/master/install.js --silent |
       PNPM_VERSION=${pnpm_ver} PNPM_DEST=${pnpm_path} PNPM_REGISTRY=${HNVM_PNPM_REGISTRY} ${node_bin} > \
       "${COMMAND_OUTPUT}"
   else
-    curl -L https://raw.githubusercontent.com/pnpm/self-installer/master/install.js |
+    curl -n -L https://raw.githubusercontent.com/pnpm/self-installer/master/install.js |
```

Comment:
auth info shouldn't be needed for curling public github content, but doesn't hurt to be consistent in how we set up auth on calls

Reply:
Yea I figure we'll need it when we curl pnpm directly instead of just the installer